src/pages/Home/Home.js

```diff
@@ -49,23 +95,30 @@ const Home = () => {
         />
       )}
       {windowType === 'delete' && <DeleteMovie closeWindow={closeWindow} />}
-      <div className={styles.search__block}>
-        <div className={styles.search__addMovieLine}>
-          <Icon />
-          <Button
-            value='+ add movie'
-            buttonClass={styles.search__addButton}
-            onClick={onAddMovieClick}
-          />
+      {!areDetailsActive && (
```

Comment:
You can use the ternary operator here:
```
movieDetailsId ? <Code /> : <AnotherCode />
```

Reply:
https://github.com/Azaliyaa/React-App/pull/3/commits/43652fd94df693efb856cb8f8273c601ddc5d200